strawberry_django/type.py

```diff
@@ -111,6 +111,14 @@ def _process_type(
             continue
         cls.__annotations__[f.name] = strawberry.auto
 
+    if is_filter:
+        cls.__annotations__.update(
+            {
+                "AND": Optional[Self],  # type:ignore
+                "OR": Optional[Self],  # type:ignore
```

Comment:
Only wondering if we should use caps names. Like, `AND`/`OR` or `and`/`or`

Personally I prefer `and`/`or`, but want to hear other opinions.

What do you think @patrick91 ?

Also, although `Self` is fine here, should we do `Optional[cls]` directly since we have it here?

Reply:
The reason I chose capitalization is to distinguish it from model fields